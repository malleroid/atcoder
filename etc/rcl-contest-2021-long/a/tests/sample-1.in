9 4 10
3 3 1 5 35
4 4 4 6 22
8 8 7 9 20
2 3 8 9 10
