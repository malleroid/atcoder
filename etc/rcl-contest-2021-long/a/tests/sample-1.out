3 3
-1
2 3
3 4
2 3 4 4
3 3 7 8
4 4 7 7
3 4 8 7
8 8
-1
